content/en/docs/concepts/security/index.md

```diff
@@ -632,331 +629,278 @@ authorization result, `ALLOW` or `DENY`.
 
 ### Enabling authorization
```

Comment:
Just call it "Implicit Enablement", to be clear. Otherwise, people may think they need to enable the feature somehow.